### PR TITLE
Updated Security Rework

### PR DESCRIPTION
**Pull request by CoZarctan**
This is Skeeve2's pr which I applied to mapmerge tools to and fixed the most obvious errors. I had to redo access levels on most things, so it's entirely possible I screwed something up there. It's probably not ready to merge yet, but I'm putting up what I've done so far.

In addition to any concerns about the raised about the design in the original forum thread (http://www.prevess.com/viewtopic.php?f=4&t=216&sid=1b72b6087265b0e60d603b0a271b9315), I noticed the following issues:

-lethal injection syringes seem to have no sprite
-solitary has no flash and no lights
-lethal injection syringes are not locked up
-cell interior/exterior blast doors have their names reversed from what I expect

Obviously a few of these are balance concerns, so I leave them up for discussion.

### CODE REVIEW

**Comment by Glloyd** [comment]
Can I just say how nice it is to have you back contributing for Urist? I'll take a look at this tomorrow or later today and we'll figure out exactly what needs to be done.

File: code/game/machinery/computer3/computers/HolodeckControl.dm [hunk 1]
```diff
@@ -90,11 +90,11 @@
...
-		else if("burntest" in href_list)
-			if(!emagged)	return
-			target = locate(/area/holodeck/source_burntest)
-			if(target)
-				loadProgram(target)
+//		else if("burntest" in href_list)
+//			if(!emagged)	return
+//			target = locate(/area/holodeck/source_burntest)
+//			if(target)
+//				loadProgram(target)
```
Comment (Glloyd) [inline code/game/machinery/computer3/computers/HolodeckControl.dm]: why is this file changed?

Comment (Skeeve2) [inline code/game/machinery/computer3/computers/HolodeckControl.dm]: This is something I accidentally left in while I was testing solutions to the holodeck fire being overpowered, I probably re-added it and saved on the same version.

Comment (CoZarctan) [inline code/game/machinery/computer3/computers/HolodeckControl.dm]: Heh yeah, we can get rid of those changes.

**Comment by Skeeve2** [comment]
Last known image of discussion : 
http://i.cubeupload.com/BpRHVU.png

Any other changes and balancing is up to you guys, have fun.

**Comment by Glloyd** [comment]
I just pushed a tiny mapmerge update, but I didn't touch sec so if you use the mapmerge tools you should be fine. That or just copy the area you've changed onto the current map, whatever you find easier.

**Comment by CoZarctan** [comment]
The mapmerge tools give me a fatal error that I don't entirely understand.
`error: addinfo_cache failed for path 'maps/Glloydstation.dmm'
Auto-merging maps/Glloydstation.dmm
maps/Glloydstation.dmm: unmerged (2be239c310684ee1bfcb76efd82b698225e62bb3)
maps/Glloydstation.dmm: unmerged (fe79b502ebf3d398ebc2eda98c4764b9ff4c6e8e)
maps/Glloydstation.dmm: unmerged (79f566f5b9fc5091333b29e8d09e280fe416a004)
fatal: git write-tree failed to write a tree
`
Some kind of permissions issue? I dunno. Anyway, other than git completely failing to recognize that a merge happened, the maps get combined just fine. Sorry about the screwed up commits. I see about resolving it when I get a chance to work on this next week (I won't be free this weekend).

**Comment by CoZarctan** [comment]
No, I guess it didn't :( I guess I really will stop fiddling around until I can do some research.

**Comment by Zuhayr** [comment]
Every time I've had that issue, I've had to kill my repo, update git and reclone to fix it.

**Comment by CoZarctan** [comment]
Yeah, I believe it may be a problem with end of line characters. I have managed to reproduce everything so far on a fresh branch (went much faster the second time). So, if anyone watching has changes to the brig map itself that they'd like to see, come up with them over the weekend, and then next week I will try to put it all together into a finalized state.

**Comment by CoZarctan** [comment]
#350
